app/generic/connections/connections.py

```diff
@@ -137,7 +139,8 @@ async def connect_to_public_did(
         their_public_did=body.public_did
     )
 
-    return conn_record_to_connection(conn_record)
+    # return conn_record_to_connection(conn_record)
+    return conn_record
```

Comment:
Why this change? We should return our connection model, not the AcaPy ConnRecord